Mechty-learn-backend/Mechty-learn-backend.sln

```diff
@@ -2,6 +2,8 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mechty-learn-backend", "Mechty-learn-backend\Mechty-learn-backend.csproj", "{2B86397B-5442-4E1D-8CBE-FBD56CC05F4F}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Mechty-learn-backend-unit-teste", "Mechty-learn-backend-unit-teste\Mechty-learn-backend-unit-teste.csproj", "{DE0C4A97-99AC-498A-BEF3-0761468248DA}"
```

Comment:
You might have a typo here: "unit-test**e**" test with an extra "e", no biggie, just letting you know.

Reply:
Yes, thank you, I noticed it already, I will fix it soon.